.eslintrc

```diff
@@ -1,14 +1,13 @@
 {
-	"extends": "eslint:recommended",
-	"plugins": ["prettier"],
+	"root": true,
+	"env": {
+		"node": true,
+		"es6": true
```

Comment:
This can probably include `browser` as a target but I'm not sure how much it matters.